multimodal/setup.py

```diff
@@ -28,6 +28,7 @@
     'requests',
     'timm<0.6.0',
     'torch>=1.9,<1.13',
+    'torchvision<0.14.0',
```

Comment:
I don't see the torchtext version bound mentioned in the PR title.

Reply:
Verifying something on CoLAB. Do not merge for now.

Reply:
I can confirm that `torchtext<0.14.0` is needed so I'm adding it back.